FirebaseCore/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
-# Unreleased
-  [changed] Directory structure reorganization without functional changes.
+# v6.6.0 -- M62
+- [changed] Reorganized directory structure.
+- [changed] The following SDKs updates introduce a new transitive dependency on the [Firebase Installations API](https://pantheon.corp.google.com/apis/library/firebaseinstallations.googleapis.com):
```

Comment:
I recommend omitting the "updates" and just say

"The following SDKs introduce [...]"